packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -56,6 +58,7 @@ import STYLES from './BpkInfoBanner.module.scss';
 const getClassName = cssModules(STYLES);
 
 const ExpandIcon = withButtonAlignment(ChevronDownIcon);
+const ContractIcon = withButtonAlignment(ChevronUpIcon);
```

Comment:
It took me a while to understand what `Contract` meant here, maybe better and what we have used in other components would be `Collapse` so keeps it consistent across Backpack, so would be 

```suggestion
const CollapseIcon = withButtonAlignment(ChevronUpIcon);
```

Reply:
Ah, yeah, I was thinking expansion and contraction, but yeah, **collapse** makes way more sense 👍